sdv/metadata/table.py

```diff
@@ -769,6 +769,8 @@ def from_dict(cls, metadata_dict, dtype_transformers=None):
             entity_columns=metadata_dict.get('entity_columns') or [],
             context_columns=metadata_dict.get('context_columns') or [],
             dtype_transformers=dtype_transformers,
+            min_value=metadata_dict.get('min_value') or 'auto',
```

Comment:
Any reason not to do

> metadata_dict.get('min_value', 'auto')

instead of this `or` syntax?

Reply:
I agree with @fealho: The `or` syntax makes sense in the cases above because it avoids having to create new `list` or `dict` objects that will end up not being used, but with the `string` this is not a problem

Reply:
Sure, will update